golang/cosmos/binding.gyp.in

```diff
@@ -35,7 +35,7 @@
           ],
           'postbuilds': [
             {
-              'postbuild_name': 'Add prefix @rpath/<(target_lib)',
+              'postbuild_name': 'Adding @rpath prefix to <(target_lib)',
```

Comment:
I don't grok this. I'll just take your word for it.

Reply:
It's just a change to a comment, resulting in the following output:

```
POSTBUILD(agcosmosdaemon): Adding @rpath prefix to libagcosmosdaemon.so
```